src/style-auro-calendar-cell.scss

```diff
@@ -0,0 +1,55 @@
+/* stylelint-disable selector-max-class, color-function-notation */
+
+.day {
+  position: relative;
+  width: var(--auro-size-600);
+  height: var(--auro-size-800);
+  border-width: 1px;
+
+  border-style: solid;
+  border-color: transparent;
+  border-radius: var(--auro-size-300);
+  background-color: transparent;
+
+  color: var(--auro-color-text-primary-on-light);
+
+  cursor: pointer;
+
+  font-family: var(--auro-font-family-default);
+  font-size: var(--auro-text-body-size-lg);
+
+  &.disabled {
+    color: var(--auro-color-text-secondary-on-dark);
+
+    pointer-events: none;
+  }
+
+  &.selected {
+    border-color: var(--auro-color-background-darker);
+
+    background-color: var(--auro-color-background-darker);
+    box-shadow: 0 .1px 5px rgba(0, 0, 0, .12), 0 5px 10px rgba(0, 0, 0, .1);
+    color: var(--auro-color-background-lightest);
+  }
+
+  &.inRange {
+    &:before {
+      position: absolute;
+      z-index: -1;
+      top: 50%;
+      left: 50%;
+      display: block;
+      width: 48px;
+      height: 48px;
+      background-color: var(--auro-color-background-lighter);
+      content: '';
+      transform: translate(-50%, -50%);
+    }
+  }
+
+  &:hover {
+    background-color: #f0f0f0; // var(--auro-color-ui-bkg-hover-on-light);
```

Comment:
???

Reply:
This is per discussion with @jason-capsule42. The token for hover has transparency. However, that causes issues with the pseudo element box on dates that are `inRange` when selecting a from and to date. We had to switch to a solid color to account for this. I left the comment on there are a sort of callout but I'll add more language to it.